Assets/Plugins/Source/Editor/EditorWindows/NewConfigWindow.cs

```diff
@@ -21,12 +21,13 @@
  */
 
 // Uncomment the following line to see the experimental new config window
-//#define ENABLE_NEW_CONFIG_WINDOW
+#define ENABLE_NEW_CONFIG_WINDOW
```

Comment:
Is this intentionally ready to be defined by default? Is maintaining this config bit a Strangler Fig thing?

Reply:
Good catch - no it should be disabled. I've fixed that.